packages/components/.storybook/config.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { Fragment } from 'react';
+
+import { withKnobs } from '@storybook/addon-knobs';
+import { withA11y } from '@storybook/addon-a11y';
+import { addDecorator, addParameters, configure } from '@storybook/react';
+import { themes } from '@storybook/theming';
+import { ThemeDecorator } from '../src/stories/decorators';
+import { createGlobalStyle } from 'styled-components';
+import global from '@codesandbox/common/lib/global.css';
+
+const viewports = {
```

Comment:
can we import this from the design language as well?

Reply:
we don't really have this there :(